server/asset/doge/doge.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/btcsuite/btcd/chaincfg"
 )
 
-var maxFeeBlocks = 50
```

Comment:
In practice, it hung and context times out with a remote node.  This just needs to make an quick effort and fall back to the "no competition" rate it can't get 101 txns in 16 blocks or less.